client/src/LoadClients/index.js

```diff
@@ -0,0 +1,73 @@
+import { useContext, useState } from 'react';
+import LoadClientContext from '../LoadClientContext';
+
+
+import "./styles.css"
+
+const LoadClients = (props) => {
+  const { loadclients, handleOpen} = useContext(LoadClientContext)
+  const [clients, setClients] = useState(loadclients);
+
+  function handleToggleSelected(id) {
```

Comment:
Just thoughts on another way to handle this. Not a fan of the index, but maybe you have some ideas to improve it.

```javascript
const updatedLoadClientIndex = clients.findIndex(lc => lc.id === id);
setClients({
    ...clients.slice(0, updatedLoadClientIndex),
    {
        ...clients[updatedLoadClientIndex],
        isExecute: !clients[updatedLoadClientIndex],
    }
    ...clients.slice(updatedLoadClientIndex + 1)
});
```

Reply:
maybe....
```javascript
const updatedLoadClient = {
    ...clients[updatedLoadClientIndex],
    isExecute: !clients[updatedLoadClientIndex],
};
```